pkg/gadgets/trace/capabilities/tracer/tracer.go

```diff
@@ -25,7 +25,7 @@ import (
 	"github.com/cilium/ebpf"
 	"github.com/cilium/ebpf/link"
 	"github.com/cilium/ebpf/perf"
-	libseccomp "github.com/seccomp/libseccomp-golang"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/utils/syscalls"
```

Comment:
I'm really sorry I forgot about this. We're trying to keep imports in blocks:
- stantard libraries
- external dependencies
- internal packages

Could you please move it to the block below where all the `github.com/inspektor-gadget/inspektor-gadget` imports are? (same for other places)